public/v4/apps/boxy-hq-only.yml

```diff
@@ -0,0 +1,252 @@
+          label: Client secret verifier
+          description: When tenant and product are used for the SAML flow (and PKCE is not being used) then we use dummy as placeholders for client_id and client_secret. This is not a security issue because SAML is tenanted and hence your Identity Provider will block access to anyone trying to log into your SAML tenant. However for additional security you should set CLIENT_SECRET_VERIFIER to a random secret and use that value as the client_secret during the OAuth 2.0 flow. (https://boxyhq.com/docs/jackson/deploy/env-variables#client_secret_verifier)
+          validRegex: '/.{1,}/'
+          defaultValue: $$cap_gen_random_hex(64)
+
+        - id: $$cap_db_engine
+          label: DB Engine
+          description: Supported values are redis, sql, mongo, mem, planetscale, dynamodb (https://boxyhq.com/docs/jackson/deploy/env-variables#db_engine)
+          validRegex: '/.{1,}/'
+          defaultValue: sql
+        - id: $$cap_db_url
+          label: DB URL
+          description: The database URL to connect to. If you are using self-signed certificates then pass sslmode=no-verify instead of sslmode=require in the DB_URL. This is because self-signed certs will be rejected as unauthorized in strict mode. Also, set DB_SSL=true and DB_SSL_REJECT_UNAUTHORIZED=false (see env vars below for more details). (https://boxyhq.com/docs/jackson/deploy/env-variables#db_url)
+          validRegex: '/.{1,}/'
+          defaultValue: postgres://postgres_user:postgres_password@localhost:5432/postgres_db
+        - id: $$cap_db_type
+          label: DB Type
+          description: Only needed when DB_ENGINE is sql. Supported values are postgres, mysql, mariadb, mssql (https://boxyhq.com/docs/jackson/deploy/env-variables#db_type)
+          validRegex: '/.{1,}/'
+          defaultValue: postgres
+        - id: $$cap_db_ttl
+          label: DB TTL
+          description: TTL for the code, session and token stores (in seconds). (https://boxyhq.com/docs/jackson/deploy/env-variables#db_ttl)
+          defaultValue: 300
+        - id: $$cap_db_cleanup_limit
+          label: DB Cleanup limit
+          description: Limit cleanup of TTL entries to this number. (https://boxyhq.com/docs/jackson/deploy/env-variables#db_cleanup_limit)
+          defaultValue: 1000
+        - id: $$cap_db_page_limit
+          label: DB Page limit
+          description: Page limit
+          defaultValue: 50
+        - id: $$cap_db_encryption_key
+          label: DB Encryption key
+          description: To encrypt data at rest specify a 32 character key. You can use openssl to generate a random 32 character key 'openssl rand -base64 24' (https://boxyhq.com/docs/jackson/deploy/env-variables#db_encryption_key)
+
+        - id: $$cap_smtp_host
+          label: SMTP Host
+          description: The SMTP host. (https://boxyhq.com/docs/jackson/deploy/env-variables#smtp_host)
+          defaultValue: smtp.example.com
```

Comment:
We should use examples as default values, move `smtp.example.com` to description.